src/python/grpcio/grpc/__init__.py

```diff
@@ -1832,6 +1832,30 @@ def local_server_credentials(local_connect_type=LocalConnectionType.LOCAL_TCP):
     return ServerCredentials(
         _cygrpc.server_credentials_local(local_connect_type.value))
 
+    
+def alts_channel_credentials(service_accounts=[]):
```

Comment:
I think it's also worthwhile to mention that ALTS credential API's can only be used in GCP environment. It is because ALTS handshake relies on a handshaker service running inside Nanny.

Reply:
Added comment for both methods with a link to ALTS documentation.